eng/feeds/arm/tspconfig.yaml

```diff
@@ -19,7 +19,7 @@ options:
   "@azure-tools/typespec-java":
     package-dir: "azure-resourcemanager-{{#toLowerCase}}{{parameters.ServiceNamespace}}{{/toLowerCase}}"
     namespace: "com.azure.resourcemanager.{{#toLowerCase}}{{parameters.ServiceNamespace}}{{/toLowerCase}}"
-    service-name: "{{#toLowerCase}}{{parameters.ServiceNamespace}}{{/toLowerCase}}"
+    service-name: "{{#toUpperCase}}{{parameters.ServiceNamespace}}{{/toUpperCase}}"
```

Comment:
Is service-name supposed to be Pascal case or upper case?

Reply:
Pascal case.

Thanks. `#toUpperCase` would be wrong. How do I write "convert to Pascal case" here? I am still not sure what's the syntax in this file.

Reply:
It's already with 'Pascal case'.  You could close this PR.
https://github.com/Azure/typespec-azure/blob/99af20a73bc3eff309ec54478d6da86f4e00ed6f/eng/feeds/azure-scaffolding.json#L240

Reply:
Unsure whether this is related.

ARM Resource Provider Name could be a same `Microsoft.Workloads` in all folders under https://github.com/Azure/azure-rest-api-specs/tree/main/specification/workloads, but each folder would generate different SDK and different service-name.

Reply:
This scaffolding template only supports a RP with single service scenario.